mlflow/models/model.py

```diff
@@ -698,3 +711,103 @@ def get_model_info(model_uri: str) -> ModelInfo:
         mlflow_version=model_meta.mlflow_version,
         metadata=model_meta.metadata,
     )
+
+
+def get_model_requirements_files(resolved_uri: str) -> Dict[str, str]:
+    requirements_txt_file = None
+    conda_yaml_file = None
+    try:
```

Comment:
Is this try-catch necesssary?

Reply:
i guess it's not necessary, we can just rely on the download functions throwing